### PR TITLE
Better errors for SSH tunnel connection failures

### PR DESCRIPTION
**Pull request by sschneid**

### CODE REVIEW

**Comment by sschneid** [comment]
@mesosphere-ci retest this please

File: shakedown/dcos/command.py [hunk 1]
```diff
@@ -39,7 +39,12 @@ def run_command(
     key = validate_key(key_path)
 
     transport = get_transport(host, username, key)
-    transport = start_transport(transport, username, key)
+
+    if transport:
+        transport = start_transport(transport, username, key)
+    else:
+        print('error: unable to connect to ' + host)
```
Comment (colin-msphere) [inline shakedown/dcos/command.py]: Please use the `"{}".format(var)` style.

File: shakedown/dcos/helpers.py [hunk 2]
```diff
@@ -29,10 +29,15 @@ def get_transport(host, username, key):
...
-           print('error: unable to authenticate ' + username + '@' + shakedown.master_ip() + ' with key ' + key_path)
-           return False
+            print('error: unable to authenticate ' + username + '@' + shakedown.master_ip() + ' with key ' + key_path)
+            return False
+
+        try:
+            channel = transport_master.open_channel('direct-tcpip', (host, 22), ('127.0.0.1', 0))
+        except paramiko.SSHException:
+            print('error: unable to connect to ' + host)
+            return False
```
Comment (colin-msphere) [inline shakedown/dcos/helpers.py]: Should this be `False` or `None`? `False` does not match the defined return type in the method docs.

File: shakedown/dcos/command.py (hunk 1 above)
Comment (sschneid) [inline shakedown/dcos/command.py]: Sure; that will make this a little bit larger of a PR but seems like a good chance to introduce more code consistency.

**Pull request merged**